spec/features/dashboard/collection_spec.rb

```diff
@@ -792,7 +788,14 @@ def get_url_fragment(type)
 
     context 'from dashboard -> collections action menu' do
       context 'for a collection' do
+        let(:collection) { build(:named_collection_lw, user: user, with_permission_template: true) }
+        let(:work1) { create(:work, title: ["King Louie"], member_of_collections: [collection], user: user) }
+        let(:work2) { create(:work, title: ["King Kong"], member_of_collections: [collection], user: user) }
```

Comment:
Moving these into the one test that uses them and removing the force load with let!, preferring a load in the before block, fixes LDP::Gone failures in tests that don't use those object. This is far less common than the other flaky test fixed by this PR.